### PR TITLE
ta - persist images to db

### LINKED ISSUE #3: Save Image Link
#### As a user I want to save a link to an image.

__Story__:

Now that we have an awesome form that can save images and display images after they have been saved. However, now would 
like to persist the saved images to our databse so that we do not lose our saved links when we reload the page.

__Acceptance criteria__:
- [x] The landing page shows the image link submission form.
- [x] The link is entered through a form.
- [x] After the form is saved, the link is persisted in the database.
- [x] After the form is submitted, the image is displayed on the page.
- [x] I cannot successfully save an image with an invalid URL on the front-end.
- [x] I cannot successfully save an image with an invalid URL on the backend.
- [x] An error message is associated with the appropriate input field on
  failure.
- [x] After the form is submitted, the image is displayed on the page and the submission form input fields are cleared.

__Discussion Topic__:
- [x] What is a valid image URL?

__Dependencies__:
- Deployment Environment

__Note__: Many image sharing sites will permit one to upload images, or
download images when provided a link. For this project we will only work with
the link, i.e., URL, to various images.

### PR DESCRIPTION
**Pull request by tyleralves**
Closes #3

### CODE REVIEW

**Comment by tyleralves** [comment]
[Ropes #3](https://trello.com/c/xJBlBhdT/610-ropes-3)

**Comment by tyleralves** [comment]
Note: Heroku failing to fetch a yarn package. Will restart tomorrow AM and hopefully it'll go through.

File: app/controllers/images_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    else
+      render json: { success: false }
+    end
+  end
+
+  private
+
+  def image_params
+    params.require(:image).permit(:url)
+  rescue ActionController::ParameterMissing
```
Comment (CRiva) [inline app/controllers/images_controller.rb]: This rescue isn't necessary, the rails model is what handles what params are valid or not.

File: app/helpers/images_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module ImagesHelper
```
Comment (CRiva) [inline app/helpers/images_helper.rb]: Since it appears you're not using this, maybe remove it till you have a PR that does use it.

Comment (tyleralves) [reply app/helpers/images_helper.rb]: Removed

File: app/controllers/images_controller.rb (hunk 1 above)
Comment (tyleralves) [reply app/controllers/images_controller.rb]: Removed

**Pull request merged**